common/main/gsw.xml

```diff
@@ -2819,9 +2819,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<displayName count="other">Maurizius-Rupie</displayName>
 			</currency>
 			<currency type="MVR">
-				<displayName>Rufiyaa</displayName>
-				<displayName count="one">Rufiyaa</displayName>
-				<displayName count="other">Rufiyaa</displayName>
+				<displayName>Malediven-Rufiyaa<</displayName>
+				<displayName count="one">Malediven-Rufiyaa<</displayName>
```

Comment:
Need to fix the << in the above two lines (should just be <)

Reply:
argh, i missed that. I'll update and squash the commits again. I need to figure out how to get better xml formatting checking on the GitHub web interface.